react-components/src/apps/details/AppInfoDialog.js

```diff
@@ -129,20 +135,15 @@ function AppInfoDialog(props) {
                 </DialogContent>
             </Dialog>
             <DEConfirmationDialog dialogOpen={confirmDialogOpen}
-                                  variant={THREE_BUTTON_VARIANT}
                                   messages={intlData}
                                   debugId={baseDebugId}
                                   onOkBtnClick={saveDoc}
-                                  onCancelBtnClick={() => setConfirmDialogOpen(false)}
-                                  onOptionalBtnClick={() => {
-                                      setConfirmDialogOpen(false);
-                                      presenter.onClose();
-                                  }}
+                                  onCancelBtnClick={onDiscardChanges}
                                   heading={formatMessage(intl, "save")}
                                   message={formatMessage(intl,"docSavePrompt")}
-                                  okLabel={formatMessage(intl, "yes")}
-                                  cancelLabel={formatMessage(intl, "cancel")}
-                                  optionalLabel={formatMessage(intl, "closeNoSave")}/>
+                                  okLabel={formatMessage(intl, "save")}
+                                  cancelLabel={formatMessage(intl, "discardChanges")}
+            />
         </React.Fragment>
     );
```

Comment:
After speaking with @psarando I modified my code to use 2 button option. I have left the code of 3 button dialog in `DEConfirmationDialog` in place in case if we need in future.